src/logger-mixin/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+export * from './mixins/logger-mixin';
```

Comment:
In #2, I laid out a directory structure where we have directories for different entities (bindings, mixins, repositories, controllers) at the first level, and then individual source files are nested in these directories. This is a convention I'd like to promote for all kinds of LoopBack-Next projects - loopback-next monorepo packages, extension packages, applications.

In that layout, your mixin file should go to `src/mixins/logger-mixin.ts`.

Here is how I envision the usage: as a developer wishing to write a new extension that provides a mixin (plus few more things like sequence action bindings), I'll rename `logger-mixin.ts` to a different name matching my extension intent (e.g. `grpc-mixin.ts`), do the same for other templates/examples I'd like to leverage (e.g. rename the template/example of a binding from. `foobar-provider.ts` to `invoke-method-provider.ts`, etc.) and have a working component.

What I'd personally like to avoid is the necessity to move files and directories around, e.g. move `src/logger-mixin/mixins/logger-mixin.ts' to `src/mixins/grpc-mixin.ts`, because it involves too many changes in other places (various `index.ts` files, related test files, etc.).